### ISSUE TITLE
(MR/VR) go full VR mode - Allow for hiding furniture or go full VR.

### ISSUE THREAD

**Issue by DaviveCZ** [fortellgames/bw_bugtracker#156]
Currently you're only able to hide real walls but can't furniture.
Some option to go full VR would be great.
![image](https://github.com/user-attachments/assets/a0938844-6bbf-490e-8190-cd423f6bdceb)

**Comment by DaviveCZ** [comment]
I posted some concerns regarding the game and compatibility with other worlds on Discord. figured i should share them here as well as this Issue/Request is regarding "Full VR" mode of some sort.

"THIS is one issue you will encounter once this game will start having world compatible with both PC, Mobile and Tablet.
Where things will become placed inside walls/furniture.
The MR stuff is great but i just cannot for the life of me see it work with worlds made on other platforms..."
"i tried playing other worlds made by other people and i encountered issue where everything was outside of my room and the only way to see it is to hide the walls but the problem is that i would still have furniture visible meaning that i won't be able to see the entirety of the world.
Again the game needs some type of Full VR mode, view rotation and teleport locomotion otherwise i just don't see this game work properly in MR/VR with other platform worlds or with worlds made with different room layout"
"like the MR possibility of this game is great but i think the game would either need to be completely retooled to fit this sort of gameplay cos this sort of thing fits Holotanks more.
but yea it's just something that's raising my concern about compatibility with other worlds.
like yea you can scale them however you want but it's not going to change the fact that every room is different layout for everyone.
I just hope some type of full VR mode with simple locomotion would be a thing. Just make sure to allow the player to rotate their view for those who play seated."

**Comment by Aktion** [comment]
We never promised full compatibility with mr/vr and other platforms, there
is a chance that MR have own store

**Comment by DaviveCZ** [comment]
ohh then that could perhaps work. But i still worry the game would encounter problem of things being inside furniture.
UNLESS! The game implements a feature similar to Holotank's remote play where it shows the other room as outline/highlight! :D

**Comment by Aktion** [comment]
We can give options how to load worlds and/or enable vr mode automatically
so furniture be ignored

**Comment by DaviveCZ** [comment]
awesome! Sorry for the concern about the VR version and compatibility, hope this feedback about it is helpful.

**Comment by Aktion** [comment]
@dianateper lets add an option to go full VR mode with removed furniture and turned off walls toggle.  
Let's add it in the options first than we will see if we can find shortcut button

**Comment by DaviveCZ** [comment]
i don't think that hiding the furniture would be enough. there needs to be some type of locomotion cos in my game i have problem where the space worlds have the blockster bay asteroid shown from outside of my room so i can't physicallly go there...

**Comment by Aktion** [comment]
We have locomotion as another task :) This one is adding Full VR into the options for now

**Comment by DaviveCZ** [comment]
I think i can close this as you can now hide furniture. and going full VR is also a thing now albeit not automatic
